tethysext/atcore/services/workflow_manager/condor_workflow_manager.py

```diff
@@ -39,9 +40,8 @@ def __init__(self, session, model_db, resource_workflow_step, user, working_dire
             jobs(list<CondorWorkflowJobNode or dict>): List of CondorWorkflowJobNodes to run.
             input_files(list<str>): List of paths to files to sends as inputs to every job. Optional.
         """  # noqa: E501
-        if not jobs or not all(isinstance(x, (dict, CondorWorkflowJobNode)) for x in jobs):
-            raise ValueError('Argument "jobs" is not defined or empty. Must provide at least one CondorWorkflowJobNode '
-                             'or equivalent dictionary.')
+        self.jobs_are_function = inspect.isfunction(jobs)
```

Comment:
Let's remove `self.jobs_are_function` and just use the `inspect.isfunction(jobs)` where needed.

Reply:
Removed self.jobs_are_funciton